### PR TITLE
Customization documentation reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: docs/admin/customize.rst [hunk 1]
```diff
@@ -70,41 +68,41 @@ Overall your module structure should look like:
...
 :file:`icons/weblate.svg`
     Logo shown in the navigation bar.
 :file:`logo-*.png`
-    Web icons depending on screen resolution and browser.
+    Web icons depending on screen resolution and web-browser.
 :file:`favicon.ico`
     Web icon used by legacy browsers.
 :file:`weblate-*.png`
-    In application avatars for bot or anonymous users. Also used by some browsers as a shortcut icon.
+    Avatars for bots or anonymous users. Some web-browsers use these as shortcut icons.
```
Comment (comradekingu) [inline docs/admin/customize.rst]: What browsers?

File: docs/admin/customize.rst [hunk 2]
```diff
@@ -70,41 +68,41 @@ Overall your module structure should look like:
...
 :file:`email-logo.png`
-    Used in notifications emails.
+    Used in notifications e-mails.
 
-And then execute :samp:`./manage.py collectstatic --noinput`, this will collect
-static files served to clients.
+3. Run :samp:`./manage.py collectstatic --noinput`, collecting static files served to
```
Comment (comradekingu) [inline docs/admin/customize.rst]:
```suggestion
3. Run :samp:`./manage.py collectstatic --noinput`, to collect static files served to
```

File: docs/admin/customize.rst [hunk 3]
```diff
@@ -116,11 +114,11 @@ static files served to clients.
...
 
-You have implemented code for :ref:`custom-autofix` or :ref:`custom-checks` and
-now it's time to install it into Weblate. First place them into your Python
-module with Weblate customization (see :ref:`custom-module`). Then enabled it
-is just matter of adding its fully-qualified path to Python class to
-appropriate settings (:setting:`CHECK_LIST` or :setting:`AUTOFIX_LIST`):
+To install your code for :ref:`custom-autofix` or :ref:`custom-checks` and in Weblate:
+
+1. Place the files in your Python module containing Weblate customization (see :ref:`custom-module`).
+2. Add its fully-qualified path to the Python class in the dedicated settings
```
Comment (comradekingu) [inline docs/admin/customize.rst]: Not sure about this. (Or the other instance below). They point to different ":setting"s too…

File: docs/admin/customize.rst [hunk 4]
```diff
@@ -70,41 +68,41 @@ Overall your module structure should look like:
         ├── addons.py
         └── checks.py
 
-You can find example application for custimizing Weblate at
-<https://github.com/WeblateOrg/customize-example>, it covers all topics
+You can find an example of for custimizing Weblate at
```
Comment (comradekingu) [inline docs/admin/customize.rst]:
```suggestion
You can find an example of custimizing Weblate at
```

File: docs/admin/customize.rst [hunk 5]
```diff
@@ -43,23 +42,22 @@ The best approach is to create a proper Python package out of your customization
         setup(
             name = "weblate_customization",
             version = "0.0.1",
-            author = "Michal Cihar",
-            author_email = "michal@cihar.com",
+            author = "Your name",
+            author_email = "yourname@example.com",
```
Comment (comradekingu) [inline docs/admin/customize.rst]: Maybe do @orangesunny, or someone not @nijel that has written a customization that made it upstream?

File: docs/admin/customize.rst (hunk 1 above)
Comment (nijel) [reply docs/admin/customize.rst]: When you create a shortcut for a website on desktop (on computer) or in the launcher (on phone).

File: docs/admin/customize.rst [hunk 6]
```diff
@@ -19,22 +18,22 @@ internal interfaces or refactoring the code.
 
 .. _custom-module:
 
-Creating Python module
+Creating a Python module
 ----------------------
```
Comment (nijel) [inline docs/admin/customize.rst]:
```suggestion
Creating a Python module
------------------------
```

File: docs/admin/customize.rst [hunk 7]
```diff
@@ -43,23 +42,22 @@ The best approach is to create a proper Python package out of your customization
...
             name = "weblate_customization",
             version = "0.0.1",
-            author = "Michal Cihar",
-            author_email = "michal@cihar.com",
+            author = "Your name",
+            author_email = "yourname@example.com",
             description = "Sample Custom check for Weblate.",
-            license = "BSD",
-            keywords = "weblate check example",
+            license = "AGPLv3+",
```
Comment (nijel) [inline docs/admin/customize.rst]:
```suggestion
            license = "GPLv3+",
```

To be consistent with Weblate. AGPL adds additional requirement for source code download, which Weblate doesn't provide.

File: docs/admin/customize.rst [hunk 8]
```diff
@@ -43,23 +42,22 @@ The best approach is to create a proper Python package out of your customization
...
+            keywords = "Weblate check example",
             packages=['weblate_customization'],
         )
 
-3. Create a folder for the Python module (also called `weblate_customization`).
-4. To make sure Python can import the module, add an :file:`__init__.py` file
-   inside the module folder. Put the rest of the customization code in this
-   folder.
-5. Now it's possible to install this package using :command:`pip install -e .`
+3. Create a folder for the Python module (also called `weblate_customization`)
```
Comment (nijel) [inline docs/admin/customize.rst]:
```suggestion
3. Create a folder for the Python module (also called ``weblate_customization``)
```

The quoting was wrong before as well.

File: docs/admin/customize.rst [hunk 9]
```diff
@@ -43,23 +42,22 @@ The best approach is to create a proper Python package out of your customization
...
+for the customization code.
+4. Within it, create a :file:`__init__.py` file to make sure Python can import the module.
+5. This package can now be installed using :command:`pip install -e<https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs>.`_
```
Comment (nijel) [inline docs/admin/customize.rst]:
```suggestion
5. This package can now be installed using :command:`pip install -e`. See more info on `editable installs <https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs>.`_
```

I don't think that what you did will work well.

File: docs/admin/customize.rst [hunk 10]
```diff
@@ -70,41 +68,41 @@ Overall your module structure should look like:
...
 
-You can find example application for custimizing Weblate at
-<https://github.com/WeblateOrg/customize-example>, it covers all topics
+You can find an example of custimizing Weblate at
+<https://github.com/WeblateOrg/customize-example>, it covers all the topics
 described below.
 
-Changing logo
+Changing the logo
 -------------
```
Comment (nijel) [inline docs/admin/customize.rst]:
```suggestion
Changing the logo
-----------------
```

File: docs/admin/customize.rst (hunk 3 above)
Comment (nijel) [reply docs/admin/customize.rst]: There is different setting for checks, fixes and addons. Maybe all these could be squashed to one chapter as they are samoe for most of the things?

File: docs/admin/customize.rst [hunk 11]
```diff
@@ -43,23 +42,22 @@ The best approach is to create a proper Python package out of your customization
...
             packages=['weblate_customization'],
         )
 
-3. Create a folder for the Python module (also called `weblate_customization`).
-4. To make sure Python can import the module, add an :file:`__init__.py` file
-   inside the module folder. Put the rest of the customization code in this
-   folder.
-5. Now it's possible to install this package using :command:`pip install -e .`
+3. Create a folder for the Python module (also called ``weblate_customization``)
+for the customization code.
```
Comment (nijel) [inline docs/admin/customize.rst]:
```suggestion
3. Create a folder for the Python module (also called ``weblate_customization``)
   for the customization code.
```

File: docs/admin/customize.rst [hunk 12]
```diff
@@ -116,11 +114,11 @@ static files served to clients.
...
-You have implemented code for :ref:`custom-autofix` or :ref:`custom-checks` and
-now it's time to install it into Weblate. First place them into your Python
-module with Weblate customization (see :ref:`custom-module`). Then enabled it
-is just matter of adding its fully-qualified path to Python class to
-appropriate settings (:setting:`CHECK_LIST` or :setting:`AUTOFIX_LIST`):
+To install your code for :ref:`custom-autofix` or :ref:`custom-checks` and in Weblate:
+
+1. Place the files in your Python module containing Weblate customization (see :ref:`custom-module`).
+2. Add its fully-qualified path to the Python class in the dedicated settings
+(:setting:`CHECK_LIST` or :setting:`AUTOFIX_LIST`) to enable it:
```
Comment (nijel) [inline docs/admin/customize.rst]:
```suggestion
2. Add its fully-qualified path to the Python class in the dedicated settings
   (:setting:`CHECK_LIST` or :setting:`AUTOFIX_LIST`) to enable it:
```

File: docs/admin/customize.rst [hunk 13]
```diff
@@ -137,9 +135,9 @@ appropriate settings (:setting:`CHECK_LIST` or :setting:`AUTOFIX_LIST`):
 Custom addons
 -------------
 
-First place them into your Python module with Weblate customization (see
-:ref:`custom-module`). Then enabled it is just matter of adding its
-fully-qualified path to Python class to appropriate settings
+1. Place them in your Python module folder containing the Weblate customization.
+(see :ref:`custom-module`).
+2. Add its fully-qualified path to the Python class in the dedicated setting section
 (:setting:`WEBLATE_ADDONS`):
```
Comment (nijel) [inline docs/admin/customize.rst]:
```suggestion
1. Place them in your Python module folder containing the Weblate customization.
   (see :ref:`custom-module`).
2. Add its fully-qualified path to the Python class in the dedicated setting section
   (:setting:`WEBLATE_ADDONS`):
```

File: docs/admin/customize.rst (hunk 7 above)
Comment (comradekingu) [reply docs/admin/customize.rst]: That doesn't have to be built in. To my knowledge network use differentiates in that it constitutes distribution, which invokes the requirement to share the code. (Per usual means.)

File: docs/admin/customize.rst (hunk 12 above)
Comment (nijel) [inline docs/admin/customize.rst]:
```suggestion
2. Add its fully-qualified path to the Python class in the dedicated settings
   (:setting:`CHECK_LIST` or :setting:`AUTOFIX_LIST`) to enable it:
```

File: docs/admin/customize.rst (hunk 7 above)
Comment (comradekingu) [reply docs/admin/customize.rst]: The problem to fix here was "BSD" not being a qualified license, as it needs to be stated how many clauses it has.

File: docs/admin/customize.rst (hunk 3 above)
Comment (comradekingu) [reply docs/admin/customize.rst]: Saw you editing, my attempt was:

> Custom quality checks, -addons and auto-fixes
---------------------------------------------

To install your code for :ref:`custom-autofix` or :ref:`custom-checks` and in Weblate:

1. Place the files in your Python module containing the Weblate customization
   (see :ref:`custom-module`).
2. Add its fully-qualified path to the Python class in the dedicated settings
   (:setting:`WEBLATE_ADDONS`), (:setting:`CHECK_LIST` or :setting:`AUTOFIX_LIST`):

.. code-block:: python

  CHECK_LIST = (
      'weblate_customization.checks.FooCheck',
  )
  
  or
 
   WEBLATE_ADDONS = (
      'weblate_customization.addons.ExamplePreAddon',
   )

.. seealso::

   :ref:`own-checks`, :ref:`own-addon`, :ref:`addon-script`

.. _custom-addon-modules:

File: docs/admin/customize.rst [hunk 14]
```diff
@@ -43,23 +42,22 @@ The best approach is to create a proper Python package out of your customization
...
 
-3. Create a folder for the Python module (also called `weblate_customization`).
-4. To make sure Python can import the module, add an :file:`__init__.py` file
-   inside the module folder. Put the rest of the customization code in this
-   folder.
-5. Now it's possible to install this package using :command:`pip install -e .`
+3. Create a folder for the Python module (also called ``weblate_customization``)
+   for the customization code.
+4. Within it, create a :file:`__init__.py` file to make sure Python can import the module.
+5. This package can now be installed using :command:`pip install -e`. See more info on `editable installs <https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs>.`_
```
Comment (nijel) [inline docs/admin/customize.rst]:
```suggestion
5. This package can now be installed using :command:`pip install -e`. See more info on :ref:`pip:editable-installs`.
```

**Comment by nijel** [comment]
Sorry, I was debugging providing feedback from documentation build and did several edits here. Please review if it still looks good. The side effect is that you can download built documentation for your changes from GitHub Actions: https://github.com/WeblateOrg/weblate/actions/runs/49630283

**Comment by comradekingu** [comment]
@nijel Looks good to me. Hoping "Enumerated list ends without a blank line; unexpected unindent." goes away following my latest attempt.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**